libvast/include/vast/system/importer.hpp

```diff
@@ -96,8 +97,30 @@ struct importer_state {
 
   std::unordered_map<caf::inbound_path*, std::string> inbound_descriptions;
 
-  measurement measurement_;
-  stopwatch::time_point last_report;
+  measurement measurement_ = {};
+  stopwatch::time_point last_report = {};
+  struct string_view_hash {
+    size_t operator()(const std::string& x) const noexcept {
+      return std::hash<std::string>()(x);
+    }
+    size_t operator()(std::string_view x) const noexcept {
+      return std::hash<std::string_view>()(x);
+    }
+  };
+  struct string_view_equal {
+    using is_transparent = void;
+    bool operator()(const std::string& lhs, std::string_view rhs) const {
+      return lhs == rhs;
+    }
+    bool operator()(std::string_view lhs, const std::string& rhs) const {
+      return lhs == rhs;
+    }
+    bool operator()(const std::string& lhs, const std::string& rhs) const {
+      return lhs == rhs;
+    }
+  };
+  tsl::robin_map<std::string, uint64_t, string_view_hash, string_view_equal>
+    schema_counters = {};
```

Comment:
I just discovered that we already have `vast/detail/heterogenous_string_hash.hpp`.
Should use that here.